editor/src/components/navigator/navigator-item/navigator-item-wrapper.tsx

```diff
@@ -25,16 +30,92 @@ interface NavigatorItemWrapperProps {
   getSelectedViewsInRange: (index: number) => Array<TemplatePath>
 }
 
+const navigatorItemWrapperSelector = createCachedSelector(
+  (store: EditorStore) => store.editor.jsxMetadataKILLME,
+  (store: EditorStore) => store.derived.canvas.transientState,
+  (store) => store.derived.navigatorTargets,
+  (store) => store.derived.elementWarnings,
+  (store, templatePath) =>
```

Comment:
the second parameter of every selector is the "props", which in our case is `templatePath`.